tfx/components/util/udf_utils.py

```diff
@@ -273,9 +273,9 @@ def package_user_module_file(instance_name: str, module_path: str,
 
   # Encode the user module key as a specification of a user module name within
   # a packaged wheel path.
-  assert '@' not in user_module_name, ('Unexpected invalid module name: %s' %
-                                       user_module_name)
-  user_module_path = '%s@%s' % (user_module_name, dist_file_path)
+  assert '@' not in user_module_name, \
```

Comment:
I'm sorry but backslash is not allowed in google style guide. Could you replace it with parentheses?

https://google.github.io/styleguide/pyguide.html#32-line-length